client/test/config/steps/password.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+let assert = require('chai').assert;
+let sinon = require('sinon');
+
+
+describe('Step: password', () => {
+
+    let jQuery;
+    let jsdom;
+    let PasswordConfig;
+    let TimeZones;
+
+
+    before(function () {
+      jsdom = require('jsdom-global')();
```

Comment:
By removing jQuery dependency I hope we will be able to also skip jsDom dependency. :)